SUTrivBot/Models/GameState.cs

```diff
@@ -0,0 +1,41 @@
+using DSharpPlus.CommandsNext;
+using DSharpPlus.Entities;
+
+namespace SUTrivBot.Models
+{
+    public class GameState : IGameState
+    {
+        private CommandContext initContext; // The command context used to initialize this game
+        private DiscordChannel channel;
+        private DiscordGuild guild;
+        private DiscordUser triviaMaster; 
```

Comment:
As before, private properties should be prefixed with "_" 
Also, Do we really need to capture the entire command context for who called it? Really we only need the DiscordUser right?

Reply:
we don't really need most of it yeah (and of course the context includes everything else already). 

I was thinking that the GameState should know what channel/guild is there in case it is needed later but it would be very easy to add later anyway. I will remove everything but the user for now.

Reply:
oh actually, I have a method in there that's using it - (so that a GameState can print a quick name for its channel/guild)